README.md

```diff
@@ -667,8 +667,9 @@ This library uses [semantic versioning](http://semver.org/). For each release, t
 * Wait for review. Once the PR is approved, merge it into `develop`
 * Fast-forward the master branch: `git checkout master && git merge --ff-only develop && git push origin master`
 * Visit [releases page](https://github.com/ably/ably-ios/releases) and `Add release notes`.
-* Release an update for CocoaPods: `(pod lib lint && pod trunk push Ably.podspec)`.
-* Generate the prebuilt framework for Carthage (`(carthage build --no-skip-current && carthage archive Ably)`) and attach the zip file to the release.
+* Release an update for CocoaPods: `pod lib lint && pod trunk push Ably.podspec`.
+* Generate the prebuilt framework for Carthage (`carthage build --no-skip-current --archive`) and attach the zip file to the release.
+* Test the integration of the library in a Xcode project using Carthage and CocoaPods.
```

Comment:
Is it obvious what to do for this?

Reply:
Well, to Cocoa developers, I think yes. But I can add more details saying to use the installation guide. Something like:

> * Test the integration of the library in a Xcode project with Carthage and CocoaPods using the [installation guide](https://github.com/ably/ably-cocoa#installation-guide).

@paddybyers Better?